tests/test_license_expression.py

```diff
@@ -645,6 +645,34 @@ def test_simplify_and_contains(self):
         expr3 = l.parse('mit and LGPL2.1')
         assert expr3 in expr2
 
+    def test_dedup(self):
```

Comment:
What would happens if I try to dedup a simple, single-key expression such as `mit` ?
Also it may be easier to read if you have several test methods for each case rather than a big one?

Reply:
Done. https://github.com/nexB/license-expression/pull/53/commits/25fa5e0d9dd15615901fe142f2589ca6890093e9